README.adoc

```diff
@@ -1,25 +1,74 @@
+
+== Starting with Spring Initializr
+
+You can use this https://start.spring.io/#!type=maven-project&language=java&platformVersion=3.0.0-RC2&packaging=jar&jvmVersion=17&groupId=com.example&artifactId=demo&name=demo&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.demo&dependencies=jdbc,cloud-task,mariadb,h2[pre-initialized project] and click Generate to download a ZIP file. This project is configured to fit the examples in this tutorial.
+
+To manually initialize the project:
+
+. Navigate to https://start.spring.io.
+This service pulls in all the dependencies you need for an application and does most of the setup for you.
+. Choose either Gradle or Maven and the language you want to use. This guide assumes that you chose Java.
+. Click *Dependencies* and select *Task*, *H2 Database*, *JDBC API*, and *MariaDB Driver*.
+. Click *Generate*.
+. Download the resulting ZIP file, which is an archive of a task application that is configured with your choices.
+
+NOTE: If your IDE has the Spring Initializr integration, you can complete this process from your IDE.
+
+NOTE: You can also fork the project from Github and open it in your IDE or other editor.
+
 
 === What is Spring Cloud Task?
 A framework for building short-lived Spring Boot microservices, such as batch data processing jobs. You can learn more about
 the framework from the link:https://spring.io/projects/spring-cloud-task[project-site],
 link:https://spring.io/projects/spring-cloud-task#learn[documentation],
 and link:https://github.com/spring-cloud/spring-cloud-task/tree/main/spring-cloud-task-samples[samples].
 
-=== Simple Task Application
-In the following guide, we develop a Spring Boot application that uses Spring Cloud Task and deploy it to Cloud Foundry,
-Kubernetes, and your local machine.
+=== Task Application Overview
+In the following guide, we develop a Spring Boot application that uses Spring Cloud Task and deploy it to your local machine.
+We will use an H2 database to initially demonstrate Spring Cloud Task functionality, but the value of Spring Cloud Task
+is in its ability to store the results of executed tasks in a persistent datastore.  For this example we will progress by using MariaDB
+as our persistent datastore.
+
+=== Spring Cloud Task using In Memory H2
+For the first step, we will use Spring Cloud Task with an embedded H2 datastore.  If no `spring.datasource` properties are detected on project
+initialization, Spring Cloud Task will spin up a new H2 instance to store the executed task results.  This happens without any extra configuration requried.
+This facilitates a
+quick way to get up and running, but the results of your executed task will be lost when the application completes and
+the H2 database shuts down.  For this step in the guide, ensure that the `spring.datasource` properties in `application.properites` are commented out.
+
+=== Spring Cloud Task using Persistent MariaDB
```

Comment:
Capitalize `using`. In title case (used for headings), every word that isn't an article, a conjunction, or a preposition (and sometimes those, too) gets capitalized.